src/components/Activity/Action.tsx

```diff
@@ -169,8 +171,8 @@ export function Action({ action, verbose }: Props) {
   const ActionComponent = ActionMapComponent[action.type]
```

Comment:
![image](https://user-images.githubusercontent.com/18054507/142929440-a1f8f920-822c-4653-afdf-bfc87f03d87f.png)

This change seems to have introduced a weird bug, can you reproduce it on your side ?